requests/utils.py

```diff
@@ -459,7 +488,7 @@ def get_encoding_from_headers(headers):
     if not content_type:
         return None
 
-    content_type, params = cgi.parse_header(content_type)
+    content_type, params = _parse_content_type_header(content_type)
```

Comment:
If `content_type` is checked above and this is a private function, do we need the guard at the top of the function above? If the check above is ever removed, we could end up with a `TypeError` when we  effectively do `content_type, params = None`.